migrations/src/postDeploymentSetup.js

```diff
@@ -127,12 +131,38 @@ module.exports = async () => {
   pendingTransactions.push(await assetManager.upgradeDelegator(delegatorAddress));
   pendingTransactions.push(await governance.grantRole(GOVERNER_ROLE, signers[0].address));
 
-  // eslint-disable-next-line no-console
   console.log('Waiting for post-deployment setup transactions to get confirmed');
   for (let i = 0; i < pendingTransactions.length; i++) {
     pendingTransactions[i].wait();
   }
 
-  // eslint-disable-next-line no-console
+  const jobs = await getJobs();
+  const collections = await getCollections();
+  console.log('Creating Jobs');
+
+  for (let i = 0; i < jobs.length; i++) {
+    const job = jobs[i];
+    await assetManager.createJob(job.weight, job.power, job.selectorType, job.name, job.selector, job.url);
+    console.log(`Job Created :  ${job.name}`);
+  }
+
+  console.log('Creating Collections');
+  console.log('Waiting for Confirm state : 4.......');
+  let state = 0;
+  const numStates = await stakeManager.NUM_STATES();
+  const stateLength = (BigNumber.from(await stakeManager.epochLength())).div(numStates);
+
+  while (state !== 4) {
```

Comment:
This entire thing can be a utility function for added readability. `await waitForConfirmState();`

Reply:
resolved 57925d7